### PR TITLE
fix: Log web console messages to file (WEBAPP-6784)

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: electron/src/main.ts [hunk 1]
```diff
@@ -544,11 +544,10 @@ class ElectronWrapperInit {
             contents.on('console-message', async (_event, _level, message) => {
               const webViewId = lifecycle.getWebViewId(contents);
               if (webViewId) {
-                const logFilePath = path.join(LOG_DIR, webViewId, config.logFileName);
                 try {
-                  await LogFactory.writeMessage(message, logFilePath);
+                  await LogFactory.writeMessage(message.replace(/%c(.+)%c /, '$1 '), LOG_FILE);
```
Comment (bennyn) [inline electron/src/main.ts:548]: Please put the regex `/%c(.+)%c` into a variable which gives information that it is about the color code.

File: electron/src/main.ts [hunk 2]
```diff
@@ -363,7 +363,7 @@ const handleAppEvents = () => {
             logger.log('Proxy prompt was submitted');
 
             const {username, password} = promptData;
-            const protocol: string | undefined = proxyInfoArg?.protocol?.replace(':', '');
+            const protocol = proxyInfoArg?.protocol?.replace(':', '');
```
Comment (bennyn) [inline electron/src/main.ts:366]: What's the intention here? Can you comment it or have a test for it?

Comment (ffflorian) [reply electron/src/main.ts:366]: The intention was to remove useless TypeScript annotations.

Comment (bennyn) [reply electron/src/main.ts:366]: I don't think that the intention of `replace(':', '')` is to remove useless TypeScript annotations or is it?

Comment (ffflorian) [reply electron/src/main.ts:366]: What do you mean? This line was added [in #3634](https://github.com/wireapp/wire-desktop/commit/dda822ed31fe636a4a70205fad898430a2ba9ee0#diff-acab454f418662fe6f697507e4c4103aR367), I just removed the type annotations.

**Pull request merged**